presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -253,6 +253,16 @@ Accessors
 
     Returns the point value that is the mathematical centroid of a geometry.
 
+.. function:: ST_Centroid(SphericalGeography) -> Point
+
+    Returns the point value that is the mathematical centroid of a spherical geometry.
+
+    It supports Points and MultiPoints as input and returns the three-dimensional centroid
+    projected onto the surface of the (spherical) Earth
+    e.g. MULTIPOINT (0 -45, 0 45, 30 0, -30 0) returns Point(0, 0))
+    Note: In the case that the three-dimensional centroid is at (0, 0, 0), the spherical centroid
+    is undefined and an arbitrary point will be returned e.g. MULTIPOINT (0 0, -180 0)
```

Comment:
This is missing the example return value?

Reply:
I meant to give only an input example, but added the actual returned value